ui/widgets/autocomplete.js

```diff
@@ -539,9 +542,10 @@ $.widget( "ui.autocomplete", {
 		// Size and position menu
 		ul.show();
 		this._resizeMenu();
+		var pos = this.isRtl() ? { my: "right top", at: "right bottom", collision: "none" } : this.options.position;
```

Comment:
@AhmedMustafa thats right for default, but not if you explicitly change the position. With what we suggested for a default of `null` by default it would show just as you describe in you pic if you did not explicitly set it. but would not override an explicitly set value.